moveit_py/test/unit/test_robot_model.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+
+from moveit_py.core import JointModelGroup, RobotModel
+
+# TODO (peterdavidfagan): depend on moveit_resources package directly.
+# (https://github.com/peterdavidfagan/moveit2/blob/moveit_py/moveit_core/utils/src/robot_model_test_utils.cpp)
+class TestRobotModel(unittest.TestCase):
+    def test_initialization(self):
+        """
+        Test that the RobotModel can be initialized with xml filepaths.
+        """
+        robot = RobotModel(
+            urdf_xml_path="./fixtures/panda.urdf", srdf_xml_path="./fixtures/panda.srdf"
```

Comment:
Consider using `os.path.join` here and in subsequent tests.

Also since it's being reused all over the place, you could probably move up the paths, or even the robot model, to a global variable.